### PR TITLE
Aditya - Changing cell node initializing value from Nan to data index

### PR DESCRIPTION
**Pull request by adiark**
NaN's were passed to initialize cell node parameter to build the graph, changed to index value 

graph.ndata['label'] = torch.tensor([-1] * num_genes + list(np.arange(len(X))))

### CODE REVIEW

**Comment by RemyLau** [comment]
Thanks for your contribution @adiark! Could you elaborate a little more on the reason for this change? Does this cause an error in one of the examples scripts you are running? If so, could you provide some more context about the error it caused and how this change fixes that particular error?

**Comment by RemyLau** [comment]
Since @WenzhuoTang worked on this module before, I'll also involve him in this thread.

**Comment by adiark** [comment]
Sure Remy. As per my understanding, on running the code graphsc.py with any example dataset, the function cell_gene_graph() was creating a graph with NAN values assigned in graph.ndata['label'] feature. So when training ID's where fetch (train_ids = np.where(labels != -1)[0]) to pass into  dgl.dataloading.NodeDataLoader() function only NaN's where passed resulting in NoneType error. Changing graph.ndata['label'] = torch.tensor([-1] * num_genes + np.nan*len(x)) to graph.ndata['label'] = torch.tensor([-1] * num_genes + list(np.arange(len(X)))), results in adding row index at the end of the tensor. Therefore, train_ids will fetch the row index to pass into NodeDataLoader function, result in creating dataloader with right trainingID's

File: dance/transforms/graph_construct.py [hunk 1]
```diff
@@ -628,7 +628,7 @@ def cell_gene_graph(data, threshold=0, dense_dim=100, gene_data={}, normalize_we
 
     graph.ndata['order'] = torch.tensor([-1] * num_genes + list(np.arange(len(X))),
                                         dtype=torch.long)  # [gene_num+train_num]
-    graph.ndata['label'] = torch.tensor([-1] * num_genes + [np.nan] * len(X))
+    graph.ndata['label'] = torch.tensor([-1] * num_genes + list(np.arange(len(X))))  #[np.nan] * len(X))
```
Comment (RemyLau) [inline dance/transforms/graph_construct.py:631]:
```suggestion
    graph.ndata['label'] = torch.tensor([-1] * num_genes + list(np.arange(len(X))))
```

**Review by RemyLau — CHANGES_REQUESTED** [review]
Thanks for the explanation. Please remove the commented part `#[np.nan]...` from the code base. Otherwise, it LGTM.

**Review by RemyLau — APPROVED** [review]

File: dance/transforms/graph_construct.py (hunk 1 above)
Comment (adiark) [reply dance/transforms/graph_construct.py]: Okay, Thank you for the review.
